ui/package.json

```diff
@@ -108,10 +110,12 @@
     "broccoli-sri-hash": "meirish/broccoli-sri-hash#rooturl",
     "codemirror": "^5.58.2",
     "columnify": "^1.5.4",
+    "d3-array": "^3.2.4",
     "d3-axis": "^1.0.8",
     "d3-ease": "^1.0.5",
     "d3-scale": "^1.0.7",
     "d3-selection": "^1.3.0",
+    "d3-shape": "^3.2.0",
```

Comment:
I removed `ember-d3` which seemed to just be bundling d3 packages. Instead, I added the d3 packages we were importing directly. (Build errors lead the way here). I wasn't sure what or where to double-check to verify removing `ember-d3` was ok.

Reply:
We will check that client counts works as expected, that's the only place d3 is used